system-addon/test/functional/mochitest/browser_highlights_section.js

```diff
@@ -0,0 +1,69 @@
+"use strict";
+
+registerCleanupFunction(async () => {
+  await clearHistoryAndBookmarks();
+});
+
+function refreshHighlightsFeed() {
```

Comment:
This looks like it could be generally useful for tests that involve highlights, so move it to head.js with docs.